### ISSUE TITLE
Double batch_process error messages for Preservica

### ISSUE THREAD

**Issue by jpengst** [yalelibrary/YUL-DC#1990]
# Summary
Some Preservica Ingest errors result in multiple error messages, with different ID's. 
Example:  
```
"Skipping row [2] SHA mismatch found in Preservica for 1 for /preservica/api/entity/information-objects/1e42a2bb-8953-41b6-bcc3-1a19c86a8y7u."
```  
Structure of error messaging should be: (reason for error, id of object that had the error). Above error has 2 separate ID's: `for 1 for /preservica/api/entity/information-objects/1e42a2bb-8953-41b6-bcc3-1a19c86a8y7u.`

**Comment by AKFosterDev** [comment]
PR Ready for review https://github.com/yalelibrary/yul-dc-management/pull/989

**Comment by martinlovell** [comment]
Released management v2.59.27
